src/librbd/io/ImageRequestWQ.h

```diff
@@ -126,10 +146,13 @@ class ImageRequestWQ
 
   bool needs_throttle(ImageDispatchSpec<ImageCtxT> *item);
 
-  void finish_queued_io(ImageDispatchSpec<ImageCtxT> *req);
+  void finish_queued_io(ImageDispatchSpec<ImageCtxT> *req, bool write_op);
+  void remove_in_flight_write_ios(uint64_t off, uint64_t len, bool write_op, uint64_t tid);
```

Comment:
Nit: pass `ImageDispatchSpec` instead of individual params

Reply:
Done